### PR TITLE
Fix database migrations

### PR DESCRIPTION
**Pull request by ZavezX**
## Description

Introduce predeployment in database migration to run before migration scripts.

- Use filesystem and path to find scripts to execute instead of embedded ressources
- Run predeployment scripts before migration scripts (These scripts must be idempotent for this to work on the next run!)
- Run migration scripts after predeployment scripts

## References

* #616

### CODE REVIEW

**Review by kristianschneider — APPROVED** [review]

File: source/coordinator/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration.csproj [hunk 1]
```diff
@@ -15,43 +15,50 @@ limitations under the License.
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="dbup" Version="4.5.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="MigrationScripts\202106191255_JobsTable.sql">
```
Comment (BjarkeMeier) [inline source/coordinator/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration.csproj:28]: Don't the scripts need to be embedded?

**Review by BjarkeMeier — COMMENTED** [review]
Should we add an integration test ensuring that migrations always work?
Or do we have enough/other tests that test the migrations implicitly?

File: source/coordinator/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration.csproj (hunk 1 above)
Comment (ZavezX) [reply source/coordinator/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration/Energinet.DataHub.Aggregation.Coordinator.DatabaseMigration.csproj:28]: The scripts don't need to be embedded, it depends on how we tell DbUp to execute scripts.

**Comment by ZavezX** [comment]
> Should we add an integration test ensuring that migrations always work? Or do we have enough/other tests that test the migrations implicitly?

We don't have any integration tests as of now to test that migrations work. But I think it's a good idea to have them. I will add a new PR with integration tests for migrations when this PR is successfully merged and deployed.

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**